plexupdate.sh

```diff
@@ -496,6 +500,11 @@ if [ "${AUTOINSTALL}" = "yes" ]; then
 	if [ ${RET} -ne 0 ]; then
 		# Clarify why this failed, so user won't be left in the dark
 		error "Failed to install update. Command '${DISTRO_INSTALL} "${DOWNLOADDIR}/${FILENAME}"' returned error code ${RET}"
+		if [ ${RET} -eq 1 ]; then
+			if [ "${DISTRO}" = "synology" || "${DISTRO}" = "synology-dsm7" ]; then
+				error "On Synology devices, you need to add Plex's public key to Package Center. If you have not done so, follow the instructions at https://support.plex.tv/articles/205165858-how-to-add-plex-s-package-signing-public-key-to-synology-nas-package-center/"
+			fi
+		fi
```

Comment:
The DSM7 package manager doesn't appear to have an interface for adding Plex's public key any more, so this may only be required for DSM6.